eogrow/utils/testing.py

```diff
@@ -266,3 +264,31 @@ def create_folder_dict(config_folder: str, stats_folder: str, subfolder: Optiona
         "config_folder": os.path.join(config_folder, subfolder) if subfolder else config_folder,
         "stats_folder": os.path.join(stats_folder, subfolder) if subfolder else config_folder,
     }
+
+
+def run_dummy_data_pipeline(
```

Comment:
I'm not entirely sure we need this utility function. We already have `run_and_test_pipeline` and you can always write a testing procedure in a way that you chain 2 pipeline configs in the same config file and then execute that with `run_and_test_pipeline`. An example of that is in `test_switch_grids.py` pipeline test. So I'm not sure when would that procedure not be enough and this function needed instead. :thinking:

Reply:
hmm, chaining configs is indeed a way to skip this :thinking: 
But would the output folder of the dummy pipeline (input folder of the proper pipeline) be *cleaned* ?

Reply:
Yes, the output of every pipeline in the chain gets removed before that pipeline is run: https://github.com/sentinel-hub/eo-grow/blob/v1.1.1/eogrow/utils/testing.py#L240

The only improvement I could think of here is that we could perhaps first delete contents of all output folders and only then run the pipeline chain. This way we also support pipelines that have same input and output folders.

Reply:
lets close this MR for now and then re-think the whole testing suite at one point